mill-db-repo/src/main/java/org/duracloud/mill/db/repo/MillJpaRepoConfig.java

```diff
@@ -91,7 +91,11 @@ public BasicDataSource millRepoDataSource() {
         emf.setPackagesToScan("org.duracloud.mill");
 
         JpaConfigurationUtil.configureEntityManagerFactory(env,emf);
-
+        if(Boolean.parseBoolean(env.getProperty("generate.database", "false"))){
+            Properties properties = new Properties();
+            properties.setProperty("javax.persistence.schema-generation.database.action", "create");
```

Comment:
The line say say:  if generate.database == "true" then create, otherwise do nothing which the default behavior.  That is 
env.getProperty("generate.database", "false")
asks for the generate.database prop and returns "false" by default.